api/sprite.md

```diff
@@ -261,7 +261,7 @@ sprite:saveAs(filename)
 ```
 
 Saves the sprite to the given file and mark the sprite as saved so
-closing it doesn't will ask to save changes.
+closing it will not ask to save changes.
```

Comment:
What do you think of using the word "won't" instead?

Reply:
Doesn't matter to me.  "won't" seems a little more casual and less formal for documentation. My personal preference would be the "will not". If you feel strongly about it I'm happy to change it!

This is also my first contribution to the project so I'm not sure if there'a style preference anything.

Reply:
My personal opinion is that "it will not ask" sounds a bit awkward, even in a formal context, because there's no need to individually emphasize the two words.
The text here doesn't sound that formal anyway.
Also, while you're at it, do you think "mark" should be changed to "marks"? (Because it is being marked)

Reply:
no prob https://github.com/aseprite/api/pull/108/commits/cd6251c669da2c380b4b8596e8f7f5a199da7de8

Reply:
Thank you very much!